tests/manage/z_cluster/nodes/test_worker_nodes_network_failures.py

```diff
@@ -0,0 +1,241 @@
+import logging
+import pytest
+from concurrent.futures import ThreadPoolExecutor
+from time import sleep
+
+from ocs_ci.framework import config
+from ocs_ci.framework.pytest_customization.marks import skipif_aws_i3
+from ocs_ci.framework.testlib import ignore_leftovers, ManageTest, tier4, tier4c
+from ocs_ci.ocs import constants, machine, node
+from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.ocs.resources import pod
+from ocs_ci.utility.utils import ceph_health_check
+from tests import helpers
+
+logger = logging.getLogger(__name__)
+
+
+@tier4
+@tier4c
+@skipif_aws_i3
+@ignore_leftovers
+class TestWorkerNodesFailure(ManageTest):
+    """
+    Test all worker nodes simultaneous abrupt network failure for ~300 seconds
+    """
+    pvc_size = 10  # size in Gi
+    num_of_app_pods = 2  # Pods of each interface type to be run
+    short_nw_fail_time = 300  # Duration in seconds for short network failure
+
+    @pytest.fixture()
+    def setup(self, request, scenario, nodes, dc_pod_factory):
+        """
+        Identify the nodes and start DeploymentConfig based app pods
+
+        Args:
+            scenario (str): Scenario of app pods running on OCS or dedicated nodes
+                (eg., 'colocated', 'dedicated')
+            nodes: A fixture to get instance of the relevant platform nodes class
+            dc_pod_factory: A fixture to create everything needed for a running
+                deploymentconfig pod
+
+        Returns:
+            tuple: containing three elements
+                obj: an instance of CephCluster class
+                list: dc pod objs
+                list: worker nodes
+        """
+        worker_nodes = helpers.get_worker_nodes()
+        ocs_nodes = machine.get_labeled_nodes(constants.OPERATOR_NODE_LABEL)
+        non_ocs_nodes = list(set(worker_nodes) - set(ocs_nodes))
+
+        def finalizer():
```

Comment:
please see my comment here https://github.com/red-hat-storage/ocs-ci/pull/2335#discussion_r453698400

Reply:
I actually see now that these fixtures are being moved to conftest.py with https://github.com/red-hat-storage/ocs-ci/pull/2492/files

Reply:
ack, will update the code